### DISCUSSION TITLE
How to query comparing field values?

### DISCUSSION THREAD

**Discussion by dfreire** [prisma/prisma#4517] [Q&A] [answered]
How can we do something like this in Prisma?
```sql
SELECT * from User WHERE firstName != lastName;
```

**Comment by NARKOZ** [comment, marked as answer]
```js
import { User } from '@prisma/client'

prisma.$queryRaw<User[]>('SELECT * from User WHERE firstName != lastName')
```

**Comment by ryands17** [comment]
Hey @dfreire 👋 

Yes this currently is only possible with a [raw query](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access) as @NARKOZ mentioned above. It would be great if you could create a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) for this to be included in the API :)

**Reply by dfreire** [reply]
[Here](https://github.com/prisma/prisma/issues/4564), thanks 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
